### ISSUE TITLE
[BUG] npm install dependency errors

### ISSUE THREAD

**Issue by RemiBauzac** [ColorlibHQ/AdminLTE#4205]
**Describe the bug**
With npm 8.3.1 / node v17.3.1, I'm facing an issue with `npm install`, because of a dependency error on `bootstrap-switch version 3.3.4`

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: admin-lte@3.2.0-rc
npm ERR! Found: bootstrap@4.6.1
npm ERR! node_modules/bootstrap
npm ERR!   bootstrap@"^4.6.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer bootstrap@"^3.1.1" from bootstrap-switch@3.3.4
npm ERR! node_modules/bootstrap-switch
npm ERR!   bootstrap-switch@"3.3.4" from the root project
npm ERR!
```

 
**To Reproduce**
1. Clone repository
2. remove package-lock.json
3. `npm install`  (with npm 8)

**Expected behavior**
Having a working npm install.

**Environment (please complete the following information):**
- AdminLTE Version: master and v3 branch
- Operating System:  Mac
- NPM (Version): 8

**Additional context**
To fix the issue, we need to change `bootstrap-switch` version from `3.3.4` to `3.4.0`, but this fix lead to huge changes on package-lock.json. How do you want to proceed ?

**Comment by REJack** [comment]
There is no fix for this since bootstrap-switch v3.4.0 isn't working, there was alot issues about this bug like this #2696.

**Comment by dlnsk** [comment]
if you use npm cli 8.3.0 or higher, add `overrides` section in your package.json to suppress dependency errors:
```
    "dependencies": {..},
    "overrides": {
        "bootstrap-switch": {
            "bootstrap": "^4.6.1"
        }
    }
```
